tests/tests/oob_indexing.rs

```diff
@@ -51,7 +51,11 @@ static RESTRICT_WORKGROUP_PRIVATE_FUNCTION_LET: GpuTestConfiguration = GpuTestCo
         drop(view);
         test_resources.readback_buffer.unmap();
 
-        assert_eq!([1; 12], current_res);
+        if ctx.adapter_info.backend == Backend::Dx12 {
```

Comment:
thought: Hmm, this is specific to FXC, per the below comment, right? Do we not have the ability to query which compiler we were using? 🤔 It might be worth getting that granularity and adding an issue for getting DXC coverage in CI.

Reply:
We don't currently have the ability to tell which shader compiler we are using. I'm hoping that in the not too distant future we will drop support for FXC as it's not worth adding and maintaining workarounds for it.